docs/using-private-template.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Using private templates"
+weight: 180
+---
+[Generator](https://www.asyncapi.com/tools/generator) allows to fetch the template from the private repositories like verdaccio, nexus, npm etc. Let's understand how can use the Generator to fetch the templates from the private repositories.
```

Comment:
```suggestion
Generator allows to fetch the template from the private repositories like verdaccio, nexus, npm etc.
```

no need to have one more sentence + link to generator - as long these are generator docs 😄

Reply:
Done @derberg